bounties/other/adevait/hrm/2/vulnerability.json

```diff
@@ -0,0 +1,54 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2020-09-13",
+    "AffectedVersionRange": "*",
+    "Summary": "XSS",
+    "Contributor": {
+        "Discloser": "b1nslashsh",
+        "Fixer": ""
+    },
+    "Package": {
+        "Registry": "other",
+        "Name": "thirtybees",
+        "URL": "https://github.com/adevait/hrm",
+        "Downloads": ""
+    },
+    "CWEs": [
+        {
+            "ID": "494",
+            "Description": ""
+        }
+    ],
+    "CVSS": {
+        "Version": "3.1",
+        "AV": "N",
+        "AC": "H",
+        "PR": "N",
```

Comment:
Hey @b1nslashsh 

I think you'll have to change this to `H` because this requires `admin` login.

And after doing those changes I think the severity will be changed to `4.0`

![image](https://user-images.githubusercontent.com/16623935/102351813-0f272000-3fcd-11eb-95d1-f5d50318d0c0.png)

Please let me know if any you feel differently

Best,
mzfr